content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst

```diff
@@ -1,7 +1,20 @@
+:nosearch:
```

Comment:
No need to include the `:nosearch:` meta tag because `removal_strategies.rst` now has content that we want searchable (instead of only a TOC). 
```suggestion
```